src/renderer/services/eth2/client/prysm/types.ts

```diff
@@ -62,3 +62,11 @@ export type ChainHead = {
     previousJustifiedEpoch: string,
     previousJustifiedBlockRoot: string
 };
+
+export type PrysmChainHeadStreamResponse = {
+    result: PrysmChainHeadResponse,
+    error: {
+        httpStatus: string;
+        message: string;
+    }
```

Comment:
Are you sure that's response?

Reply:
Says so: https://api.prylabs.net/#/BeaconChain/StreamChainHead
Unfortunately I couldn't see for myself :D

Reply:
Oh, rename this to PrysmChainHeadStreamMessage since Chainhead is this:
https://api.prylabs.net/#/BeaconChain/GetChainHead